docs/hotkeys.md

```diff
@@ -13,6 +13,7 @@
 |Redraw screen|<kbd>Ctrl</kbd> + <kbd>l</kbd>|
 |Quit|<kbd>Ctrl</kbd> + <kbd>c</kbd>|
 |Show/hide user information (from users list)|<kbd>i</kbd>|
+|Next footer hint|<kbd>Tab</kbd>|
```

Comment:
Sure, we could totally go with that. That would be better indeed, at this point.
I was actually thinking of adding functionality to go back to the previous hint as well, but it's too early to decide, I suppose, as it would depend on other features that are yet to be implemented like the hint priority field.

Reply:
Oh, that makes sense. In that case, it's probably best to keep it as is.

Reply:
Nah, I've added a new commit updating it already :)
We could change it to "Next" if and when we add the functionality for going to previous hint.